### PR TITLE
improving shogun documentaiton

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
-:sup:`(*)` Remember that you can also upload BIOM tables for plotting but not covered here because this is only for raw data.
 
 Target gene barcoded sequencing
 -------------------------------
 
-For this you can start with raw, not demultiplexed data or per_sample_FASTQ, see :ref:`example_study_processing_workflow`. Either way, you will need to
-"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created you need to select which processing to perform.
-There are two main ideologies/methodologies to process target gene data: sequence clustering and sequence cleanup.
+For this you can start with raw, not demultiplexed data or per_sample_FASTQ. Either way, you will need to
+"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created
```
Comment (qiyunzhu) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: Missing period.

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
-
-:sup:`(*)` Remember that you can also upload BIOM tables for plotting but not covered here because this is only for raw data.
 
 Target gene barcoded sequencing
 -------------------------------
 
-For this you can start with raw, not demultiplexed data or per_sample_FASTQ, see :ref:`example_study_processing_workflow`. Either way, you will need to
-"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created you need to select which processing to perform.
-There are two main ideologies/methodologies to process target gene data: sequence clustering and sequence cleanup.
+For this you can start with raw, not demultiplexed data or per_sample_FASTQ. Either way, you will need to
```
Comment (qiyunzhu) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: How about "pre-demultiplexing", "per-sample FASTQ"?

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 3]
```diff
@@ -1,32 +1,25 @@
...
+"Split libraries and QC", which uses the default in QIIME 1.9.1. Once your demultiplexed and QCed artifact is created
+you need to select which processing to perform. There are two main ideologies/methodologies to process target
+gene data: sequence clustering and sequence cleanup.
```
Comment (qiyunzhu) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: I noticed that you replaced "cleanup" with "deblur" below. Should this place be also edited for consistency?

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 4]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
-Next, the QC'd sequences will be compared against reference databases to determine the presence and abundance of protein-coding functional genes and
-pathways using HUMAnN2. These are then summarized as BIOM tables, which can be used in subsequent analysis and visualization.
-
-For more information visit the `Shotgun Qiita Plugin GitHub page <https://github.com/qiita-spots/qp-shotgun>`_.
+Qiita currently has one shotgun metagenomics data analysis pipeline: `Shogun <https://msystems.asm.org/content/3/6/e00069-18>`_.
+
+The current workflow is as follows:
+
+#. Removal of adapter sequence and quality control: `Atropos <https://github.com/jdidion/atropos/>`_
+#. Host contamination using `Bowtie2 <http://bowtie-bio.sourceforge.net/bowtie2/index.shtml>`_
```
Comment (qiyunzhu) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: Add "Removal of" to the beginning?

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 5]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
+#. Taxonomy profiling using three different aligners and two different reference databases; see sections below
```
Comment (qiyunzhu) [inline qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: How about "...using **choice of** three different..."

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 2 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 1 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: not sure which one

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 3 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 5 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 4 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 6]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
+
+Note that we recommend only uploading sequences that have already been through QC and human sequence removal. However, we
+recommend that all sequence files go through adapter and quality control within the system to ensure they are ready for
+subsequent analyses. Currently, the command removes adaptor sequences (only KAPA HyperPlus with iTru).
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: good to know! thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 7]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
+
+For host removal we currently support Danio Rerio, Drosophila Melanogaster, Mus Musculus, PhiX, and Rattus Norvegicus.
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: sound good!

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 8]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
+
+Note that the Shogun command produces 4 output artifacts:
+- The Alignment Profile BIOM artifact, which contains the alignment files
+- A Taxonomic Prediction - phylum BIOM artifact, which contains the taxonomic predictions based on the alignment
+- A Taxonomic Prediction - genera BIOM artifact, which contains the taxonomic predictions based on the alignment
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst [hunk 9]
```diff
@@ -43,23 +36,95 @@ Here we generate a single BIOM table with the OTUs/per-sample. The OTU IDs are g
...
+
+     - Kingdoms: 2
+     - Phyla: 146
+     - Classes: 89
+     - Orders: 196
+     - Families: 422
+     - Genera: 2,081
+     - Species: 9,105
+     - Strains: 89
+     - Note: Nucleotide sequences per genome were concatenated with a linker of 20 "N"s.
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: The over identation of the block is for nice formatting ...

File: qiita_pet/support_files/doc/source/resources.rst [hunk 10]
```diff
@@ -26,8 +26,8 @@ Shotgun Sequencing
 ^^^^^^^^^^^^^^^^^^
 
 - `Plugin code <https://github.com/qiita-spots/qp-shotgun>`__
-- Main resource: `HUMAnN2 <https://bitbucket.org/biobakery/humann2/wiki/Home/>`__
-- Status: In development
+- Main resource: `Shogun <https://msystems.asm.org/content/3/6/e00069-18>`__
+- Status:Fully deployed
```
[the remark this answers is not in the document]
Comment (antgonza) [reply qiita_pet/support_files/doc/source/resources.rst]: thanks

File: qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst (hunk 1 above)
Comment (qiyunzhu) [reply qiita_pet/support_files/doc/source/processingdata/processing-recommendations.rst]: Sorry! My mis-catch.

**Pull request merged**